google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -1334,6 +1393,12 @@ def create_file file, path = nil, acl: nil, cache_control: nil,
         #   used. All source files must have been encrypted with the same key,
         #   and the resulting destination file will also be encrypted with the
         #   key.
+        # @param [Integer] if_generation_match Makes the operation conditional
```

Comment:
I would call out that this parameter is for the destination object name.

Reply:
Done, thanks!